quattor/types/aquilon/system.pan

```diff
@@ -55,9 +55,11 @@ type structure_archetype_os = {
 };
 
 type structure_archetype = {
-    "name"          : string # e.g. "aquilon"
+    "name"          ? string # e.g. "aquilon"
```

Comment:
should be mandatory to match ours.

Reply:
I agree, it should... But as of Aquilon 1.12.60 it is not added to the hostdata by the broker... Should be fixed IMO... See https://github.com/quattor/aquilon/issues/90